starlette/testclient.py

```diff
@@ -18,6 +19,11 @@
 from starlette.types import Message, Receive, Scope, Send
 from starlette.websockets import WebSocketDisconnect
 
+if sys.version_info >= (3, 8):  # pragma: no cover
+    from typing import TypedDict  # pragma: no cover
+else:  # pragma: no cover
+    from typing_extensions import TypedDict  # pragma: no cover
```

Comment:
Think these are only needed on the if/elses?
```suggestion
if sys.version_info >= (3, 8):  # pragma: no cover
    from typing import TypedDict
else:  # pragma: no cover
    from typing_extensions import TypedDict
```

Reply:
let's see what happens